clouddriver-kubernetes/src/main/groovy/com/netflix/spinnaker/clouddriver/kubernetes/v1/api/KubernetesClientApiAdapter.groovy

```diff
@@ -172,7 +172,7 @@ class KubernetesClientApiAdapter {
         }
 
         return list.items
-      } catch (ApiException ex) {
+      } catch (ApiException e) {
```

Comment:
Can you pull this fix into a separate PR that we can merge into the release-1.5.x branch as well?

Reply:
@lwander will do it.